CHANGELOG.rst

```diff
@@ -22,14 +22,15 @@ Features
 * Added ``mean_tracer_den_unit`` -- the mean density of whatever tracer you use.
 * Interfaces with COLOSSUS for concentration and bias.
 * New CLI interface ``halomod run``
-* New examples, including an example of parameter fitting with ``halomod`` and ``emcee`.
+* New examples, including an example of parameter fitting with ``halomod`` and ``emcee``.
 * New hod, density profile and concentration model, with new example, for HI.
 
 Changes
 +++++++
 
 * Removed dynamic ``mmin``.
 * No more support for Python 2.
+* Update tutorial to match the current version.
```

Comment:
This should be in a new "dev-version" section at the top (2.0.0 has been released). This also means that the version printed out in the Jupyter notebook should be v2.0.0.